src/components/HeadCommon.astro

```diff
@@ -41,3 +41,34 @@ import "@chainlink/design-system/global-styles.css"
   })(window, document, "script", "dataLayer", "GTM-N6DQ47T")
 </script>
 <!-- End Google Tag Manager -->
+
+<!--Start Weglot Script-->
+<link rel="alternate" hreflang="en" href="https://www.docs.chain.link.com" />
+<script type="text/javascript" src="https://cdn.weglot.com/weglot.min.js"></script>
+<script>
+  if (
+    // Run this script only on Vercel prod and preview
+    window.location.hostname.includes("docs.chain.link") ||
+    window.location.hostname.includes("documentation-git-")
+  ) {
+    Weglot.initialize({
+      api_key: "wg_bc56a95905bfa8990f449554339e82be8",
+      switchers: [
+        {
+          button_style: {
+            full_name: false,
+            with_name: true,
+            is_dropdown: true,
+            with_flags: false,
+            flag_type: "circle",
+          },
+          location: {
+            target: "body > header",
```

Comment:
Update the header so that there is an empty div next to the github icon
![CleanShot 2023-06-16 at 09 18 06@2x](https://github.com/smartcontractkit/documentation/assets/16008141/880b952b-0bd9-459e-bc3e-de069011dbbd)

Then you can update the location target so that the dropdown gets inserted inside the navbar

Reply:
Done